README.md

```diff
@@ -77,6 +79,10 @@ You can set the proxy url by either setting `OVERRIDE_API_PROXY` or `API_PROXY`:
 API_PROXY=http://localhost:8888 npm start
 ```
 
+### Try it out
+
+Try out DEAR Petition Generator by logging in as the superuser you created and uploading test CIPRS records (in pdf format) which can be downloaded from https://github.com/deardurham/ciprs-reader/tree/main/tests/test_records
```

Comment:
Thanks for adding this. I'm not entirely sure where the proper place is to add it. I'm thinking the `Quick Start` section because that's likely the section most new developers would be reading

Reply:
Will do